--- a/src/game/prop.c
+++ b/src/game/prop.c
@@ -824,17 +824,17 @@
     }
 #endif
 
-    if (arg1->pad >= 0)
+    if (arg1->lookpad >= 0)
     {
         temp_a2 = (struct coord3d*)arg1->model->obj->Switches[0]->Data;
 
-        if (isNotBoundPad(arg1->pad))
+        if (isNotBoundPad(arg1->lookpad))
         {
-            sp50 = &g_CurrentSetup.pads[arg1->pad];
+            sp50 = &g_CurrentSetup.pads[arg1->lookpad];
         }
         else
         {
-            sp50 = (struct PadRecord *)&g_CurrentSetup.boundpads[getBoundPadNum(arg1->pad)];
+            sp50 = (struct PadRecord *)&g_CurrentSetup.boundpads[getBoundPadNum(arg1->lookpad)];
         }
 
         sp44.f[0] = temp_a2->f[0];
